packages/xcoobee-cookie-kit-react/src/BridgeCommunicator.js

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { xbOrigin } from "./configs";
```

Comment:
I don't see an `xbOrigin` in `src/configs.js`. Did you mean `xbApiUrl`?

Reply:
You are looking at `xcoobee-cookie-kit-core/src/configs.js` file instead of `xcoobee-cookie-kit-react/src/configs.js`.